change/@fluentui-react-charting-0091dd21-21f9-4537-8fd9-fbfee2e292e7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Integrate microsoft/monosize into react-charting",
```

Comment:
update to mention the charts for which this has been added. Add a chore: suffix in the beginning of the comment

Reply:
Done